index.html

```diff
@@ -0,0 +1,1048 @@
+    <link rel="stylesheet" href="./public/animate.css" />
+    <link rel="stylesheet" href="./public/style.css" />
+  </head>
+  <body>
+    <header class="header list">
+      <a href="./index.html" class="header__logo">
+        <h1 class="header__title">
+          <img
+            src="./images/icons/logo.svg"
+            alt="Pixmart logo"
+            width="108"
+            height="24"
+          />
+        </h1>
+      </a>
+      <nav class="header__navigation">
+        <ul class="navigation__list list">
+          <li class="navigation__list-item">
+            <a
+              href="#"
+              class="navigation__link button button__small button__navigation"
+            >
+              About
+            </a>
+          </li>
+          <li class="navigation__list-item">
+            <a
+              href="./catalog.html"
+              class="navigation__link button button__small button__navigation"
+            >
+              Products
+            </a>
+          </li>
+          <li class="navigation__list-item">
+            <a
+              href="#"
+              class="navigation__link button button__small button__navigation"
+            >
+              Creators
+            </a>
+          </li>
+          <li class="navigation__list-item">
+            <a
+              href="./blog-details.html"
+              class="navigation__link button button__small button__navigation"
+            >
+              Blog
+            </a>
+          </li>
+          <li class="navigation__list-item">
+            <a
+              href="./contacts.html"
+              class="navigation__link button button__small button__navigation"
+            >
+              Contact
+            </a>
+          </li>
+        </ul>
+      </nav>
+      <div class="header__assets-block list">
+        <form action="#" method="get" class="assets-block__searching">
+          <label for="asset-searching" class="searching__label hidden"
+            >Find asset</label
+          >
+          <input
+            type="text"
+            class="searching__input input input__small"
+            id="asset-searching"
+            placeholder="Search for Digital Assets"
+          />
+          <button type="submit" class="searching__button hidden">Search</button>
+        </form>
+        <a
+          href="#"
+          class="assets-block__button button button__primary button__square-medium"
+        >
+          <svg class="assets-block__icon icon">
+            <use xlink:href="./images/icons/sprite.svg#bag"></use>
+          </svg>
+        </a>
+      </div>
+    </header>
+    <main class="container__main">
+      <section class="main">
+        <div class="main__main-information list list__column">
+          <h2 class="main-information__title hidden heading__section">
+            Introduction
+          </h2>
+          <strong class="main-information__highlite-text heading__main-title">
```

Comment:
The [strong](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-strong-element) element can be used in a heading, caption, or paragraph to distinguish the part that really matters from other parts that might be more detailed, more jovial, or merely boilerplate.

Following this logic it's a Paragraph(P) tag with highlighted part("Digital") via STRONG tag.

Reply:
Fixed 🥇